mlflow/protos/databricks_uc_registry_messages.proto

```diff
@@ -190,6 +192,26 @@ enum StorageMode {
   DEFAULT_STORAGE = 2;
 }
 
+message EncryptionDetails {
+  oneof encryption_details_type {
+    // Details for CLOUD_MANAGED_SSE_KEYS:
```

Comment:
`oneof` just one option? I assume we're trying to match some passed in format exactly, so if this is necessary, let's add a comment explaining why

Reply:
This is to match the UCMR proto, derived from the UC proto. We're only exposing the information we need.